discord/state.py

```diff
@@ -1383,18 +1592,87 @@ def _get_create_guild(self, data):
     def is_guild_evicted(self, guild) -> bool:
```

Comment:
What does `is_guild_evicted` mean?

Reply:
IIRC it's if a guild can be accessed (i.e. it isn't just received from an old fetch of a guild you've left or something).

Reply:
Just added back the check. Accidentally removed it.